.github/workflows/amazon-cloudwatch-observability-image-scan.yaml

```
@@ -0,0 +1,84 @@
+name: Run Image Scan for Amazon CloudWatch Observability Helm Chart
+
+on:
+  schedule:
+    - cron: 0 13 * * 1 # Every Monday at 1PM UTC (9AM EST)
```

nit: Feels a bit more readable if we use the day name instead.

```suggestion
    - cron: 0 13 * * MON # Every Monday at 1PM UTC (9AM EST)
```